ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml

```diff
@@ -864,7 +864,7 @@ tests:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-baselinecaps-v412
 - as: azure-ipi-baselinecaps-v412-p3-f28-destructive
-  cron: 16 18 2 * *
+  cron: 16 18 4 * *
```

Comment:
Isn't it already executed ?

Reply:
But executed failed, so re-trigger again

Reply:
We should use rebuild in that case.

Reply:
okay, got it, thanks, updated